src/containers/AppSetting/AppSetting.tsx

```diff
@@ -0,0 +1,127 @@
+import { Col, Container, Row } from 'react-bootstrap';
+import { useTitle } from '@/hooks/useTitle';
+import { ROUTE_KEY, getRouteByKey } from '@/routes/routeConfig';
+
+import { Card, CardBody, CardTitleWrap, CardTitle, CardSubhead } from '@/shared/components/Card';
+import {
+  FormContainer,
+  FormGroup,
+  FormGroupField,
+  FormGroupLabel,
+} from '@/shared/components/form/FormElements';
+import { Controller, useForm } from 'react-hook-form';
+
+import { useUserContext } from '@/hooks/userHooks';
+import { THEME } from '@/shared/constants/storage';
+
+const AppSetting = () => {
+  const {
+    control,
+    // formState: { errors },
+  } = useForm();
+
+  const { store, setStore } = useUserContext();
+  const themeColor = store.themeColor || localStorage.getItem(THEME) || 'dark';
+
+  const changeTheme = () => {
+    //TO DO: CP-33
+    setStore({});
+  };
+
+  //Testing array, to be removed after implement the real language switching function
+  const languagesList: String[] = ['English', 'Chinese', 'Japanese', 'Korean'];
+
+  useTitle(getRouteByKey(ROUTE_KEY.APPSETTING).title);
+
+  return (
+    <Container>
+      {/*  
+      <Row>
+        <Col md={12}>
+          <h3 className='pape-title'>{document.title}</h3>
+          <h3 className="pape-title">AppSetting</h3>
+        </Col>
+      </Row>
+      */}
+
+      <Row>
+        <Col md={12} lg={12}>
+          <Card>
+            <CardBody>
+              <CardTitleWrap>
+                <CardTitle>App Settings</CardTitle>
+                <CardSubhead>Change your app settings</CardSubhead>
+              </CardTitleWrap>
+              <FormContainer $horizontal>
+                <FormGroup>
+                  <FormGroupLabel>Language</FormGroupLabel>
+                  <FormGroupField>
+                    <Controller
+                      name="Language"
+                      control={control}
+                      rules={{ required: 'Theme selection is required' }}
+                      defaultValue={'English'}
```

Comment:
Curly braces are unnecessary here.

Reply:
compiler report bug after remove the inner curly braces
<img width="765" alt="Screenshot 2024-04-19 at 7 02 32 AM" src="https://github.com/BeeQuantAI/platform_app/assets/84086904/6361e063-68fd-42bc-92b6-65854f840c3e">

Reply:
Curly braces on ` defaultValue`